src/commands/towerverse/cmd.online.ts

```diff
@@ -0,0 +1,30 @@
+// License: GPL-3.0
+// Maintainer(s): TheOtterlord, KittyBorgX
+
+import { MessageEmbed } from "discord.js";
+import { Client } from "towerverse.js";
+import { cooldown } from "../../middleware/cooldown";
+import { Command } from "../../structures/Command";
+
+const command = new Command({
+  name: "online",
+  desc: "Shows number of online travellers",
+  args: [{ name: "server", required: false }],
+});
```

Comment:
this argument is not used
```suggestion
  desc: "Shows number of online travellers",
});
```

Reply:
done